src/libraries/sendtohelixhelp.proj

```diff
@@ -233,13 +232,46 @@
     </ItemGroup>
   </Target>
 
+  <PropertyGroup>
+    <_ShouldSetXUnitLogCheckerEnvVar>0</_ShouldSetXUnitLogCheckerEnvVar>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(IsXUnitLogCheckerSupported)' == 'true'">
+    <_ShouldSetXUnitLogCheckerEnvVar Condition="$(TargetFrameworkIdentifier) == '.NETCoreApp'">1</_ShouldSetXUnitLogCheckerEnvVar>
```

Comment:
TargetFrameworkIdentifier isn't defined in this project. Use BuildTargetFramework instead.

Reply:
Done: $(BuildTargetFramework) == '$(NetCoreAppCurrent)'